src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -39,6 +39,12 @@ $bcs-task-left-offset: 10 * $bcs-task-grid + 2px;
 
 .bcs-Task-statusContainer {
     margin-top: 2 * $bcs-task-grid;
+    margin-bottom: 10px;
+}
+
+.bcs-Task-dueDateContainer {
+    margin-bottom: 13px;
+    text-transform: uppercase;
```

Comment:
Capitalizing whole due date. Do not want the `at` to be capitalized, however not sure how to go about this bc of how `ReadableTime` is laid out. @alexkrolick

Reply:
That will really vary across languages. Hard to generalize that design.

Reply:
Yes, we don't want to apply this after the fact; this would break arabic, and wouldn't apply to any CJK languages in a meaningful way. Distinguish these from non-uppercase as new strings (some of which I added already)

cc @ehoogerbeets

Reply:
Confirmed w/ design that all uppercase is OK.

Reply:
@il10, it looks like @booc0mtaco's concern is still valid?

Reply:
@jstoffan @booc0mtaco removing `text-transform` and leaving text as is for now. Will make changes to providing uppercase in a follow up PR if permitted by Moji team.